app/plugins/text-mining/text-mining.html

```diff
@@ -1,5 +1,20 @@
 <!-- plugin template -->
 <div ng-show="ext.data.length>0">
+
+    <div ng-show="ext.nferx.data.length>0">
+
+        <div class="clearfix">
+
+            <p>Source: Nferx
+                <ot-nferx
+                        target="{{target.id}}"
+                        disease="{{disease.efo}}"
+                        ext="ext.nferx">
+                </ot-nferx>
+            </p>
+        </div>
+    </div>
+
```

Comment:
@emepyc @LucaFumis @elipapa -There is a separate directive for nferx - nferx-directive.js. But I call this directive from the text-mining html- do you think we should use with this approach? Or  have a new 'Nferx' plugin that includes both the text mining and nferx related directives - it stays part of public repo . In the public version -evidenceSection will point to original text-mining plugin and in the Biogen private version - evidenceSection will override it with the new Nferx plugin?